Src/EmbeddedScripts.CSharp.MonoEvaluator/ReflectionImporterExtension.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Linq;
+using System.Reflection;
+using Mono.CSharp;
+
+namespace EmbeddedScripts.CSharp.MonoEvaluator
+{
+    public static class ReflectionImporterExtension
+    {
+        public static bool ContainsAssembly(this ReflectionImporter importer, Assembly assembly)
```

Comment:
nit: as it's used in a 1 single place, it can be an EvaluatorExtension instead, then you don't see internal type ReflectionImporter in the MonoCSharpRunner code